Makefile

```diff
@@ -1,8 +1,8 @@
 SHELL = /bin/bash
 
 build:
-	rubocop -a && \
-	bundle install
+	bundle install && \
+	rubocop -a
```

Comment:
This works better if `bundle install` has never been called on the project (as I learned after reinstalling Ruby)